### PR TITLE
WooCommerce sticky header: stick to top, make 100% width

### PR DESCRIPTION
**Pull request by kellychoffman**
This is a /try branch because I wasn't 100% sure it would work with only style changes. Turns out, it got pretty close. 

Added @coderkevin as you originally tried to do this and I think have on your to-do list.

To test, go to `http://calypso.localhost:3000/store/products/SITEURL/add` and verify there's a sticky header with breadcrumbs, save, and delete. 

Before:
screen shot 2017-06-06 at 3 48 04 pm

After: 
screen shot 2017-06-06 at 3 47 11 pm

Note: This only works on the product page. When @coderkevin hooks up the breadcrumbs navigation, this will have to be added to the rest of the pages.

### CODE REVIEW

**Comment by jameskoster** [comment]
It might just be me but I feel the 'weight' of the element isn't matching it's significance/importance in the UI.

I enlarged everything a bit in the Inspector for a quick comparison;

bigger-bar

What do you think?

It bugs me a little how the shadow color doesn't quite match the border on `store-sidebar__ground-control`. But I like the visual effect of it as you scroll. Minor point, probably not much we can do.

**Comment by kellychoffman** [comment]
I agree. I'll work on the fix.

**Comment by kellychoffman** [comment]
Okay, updated the height. Feels better now. 

The only thing I'm not sure about is how to update the sidebar loading indicator to match the new height.

**Comment by kellychoffman** [comment]
We have to get the back button to display in the mobile view, like it does in the editor.

Right now:
now

Editor:
editor

@justinshreve: is this something you could help us out with? 😁

**Comment by kellychoffman** [comment]
Okay, this is ready for review again. There is now a `<` button to access the sidebar on smaller screens in the sticky header.

File: client/components/sticky-panel/style.scss [hunk 1]
```diff
@@ -1,7 +1,28 @@
+.sticky-panel {
```
Comment (coderkevin) [inline client/components/sticky-panel/style.scss]: Is it okay to modify this Calypso component directly? Has anyone checked to see the impact to the rest of Calypso? We probably need a checklist of all the places it's used so we can check them before merging.

Comment (kellychoffman) [inline client/components/sticky-panel/style.scss]: No it is not. Thanks for catching that. I'll fix.

Comment (kellychoffman) [inline client/components/sticky-panel/style.scss]: Fixed this in 8a21c7e.

**Comment by kellychoffman** [comment]
@allendav - How do I adjust the site indicator loading animation in the sidebar to match the new height?

**Comment by allendav** [comment]
@kellychoffman when the site is not-yet-available, `div.store-sidebar__ground-control-site` is empty - if you set a `min-height` on that it should take care of things

**Comment by kellychoffman** [comment]
> Obviously we need to add this to all other pages but let's do that in another PR.

Yes, that and hook up the breadcrumbs, which we should probably do before we add it to all the pages. @coderkevin: Is the breadcrumbs something you can look at? I can add another issue if you don't think we should add it to this PR.

**Comment by coderkevin** [comment]
@kellychoffman I hadn't planned on working on breadcrumbs this week. Didn't @allendav do some work on that a couple weeks back? Does anyone know what the current status of it is?

**Comment by kellychoffman** [comment]
If there's no time for v1, we can agree to just display the current page, ie `Products`.

File: client/components/sticky-panel/style.scss [hunk 2]
```diff
@@ -11,4 +11,4 @@
 	.sticky-panel.is-sticky .sticky-panel__content {
 		margin-top: 8px;
 	}
-}
\ No newline at end of file
+}
```
Comment (justinshreve) [inline client/components/sticky-panel/style.scss]: I know this is just fixing a missing newline that all files should have (and was probably done automatically by your editor) but we should probably drop this change to keep all changes in this PR in the woocommerce directory.

**Pull request merged**